### PR TITLE
[one-cmds] Add ODEFAULT.cfg file for optimization

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's add ODEFAULT.cfg file for optimization.

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/CMakeLists.txt [hunk 1]
```diff
@@ -40,7 +40,7 @@ foreach(ONE_COMMAND IN ITEMS ${ONE_COMMAND_FILES})
                       GROUP_READ GROUP_EXECUTE
                       WORLD_READ WORLD_EXECUTE
           DESTINATION bin)
-          
+
```
Comment (YongseopKim) [inline compiler/one-cmds/CMakeLists.txt:43]: Trim whitespace

**Comment by YongseopKim** [comment]
build/install

```
build/install $ ls
bin  doc  include  lib  optimization  test  test.log

build/install $ ls optimization
ODEFAULT.cfg

build/install $ cat optimization/ODEFAULT.cfg
# These options copied from 'onelib/constant.py`
[one-optimize]
; Experimental: This will convert NCHW operators to NHWC under the assumption that input model is NCHW.
convert_nchw_to_nhwc=True
; expand broadcastable constant node inputs
expand_broadcast_const=True
; onvert the input shape of the model (argument for convert_nchw_to_nhwc)
nchw_to_nhwc_input_shape=True
; convert the output shape of the model (argument for convert_nchw_to_nhwc)
nchw_to_nhwc_output_shape=True
; fold AddV2 op with constant inputs
fold_add_v2=True
...
```

**Comment by YongseopKim** [comment]
onecc -h
```
$ onecc -h
usage: onecc [-h] [-v] [-C CONFIG] [-O OPTIMIZATION] [COMMAND <args>]

Run ONE driver via several commands or configuration file

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
  -V, --verbose         output additional information to stdout or stderr
  -C CONFIG, --config CONFIG
                        run with configuation file
  -O OPTIMIZATION       optimization name to use (Available optimization options: -ODEFAULT)
```

**Comment by YongseopKim** [comment]
```
$ cat mycfg.cfg
[onecc]
one-optimize=True

[one-optimize]
input_path = inception_v3_tflite.circle
output_path = inception_v3_tflite.opt.circle
O = DEFAULT
```

and then, run `onecc -C mycfg.cfg`
```
$ cat inception_v3_tflite.opt.circle.log
/home/dragon/Works/ONE/build/install/bin/circle2circle inception_v3_tflite.circle inception_v3_tflite.opt.circle --convert_nchw_to_nhwc --expand_broadcast_const --nchw_to_nhwc_input_shape --nchw_to_nhwc_output_shape --fold_add_v2 --fold_cast --fold_dequantize --fold_dwconv --fold_gather --fold_sparse_to_dense --forward_reshape_to_unaryop --fuse_add_with_tconv --fuse_add_with_fully_connected --fuse_batchnorm_with_conv --fuse_batchnorm_with_dwconv --fuse_batchnorm_with_tconv --fuse_bcq --fuse_preactivation_batchnorm --fuse_mean_with_mean --fuse_transpose_with_mean --make_batchnorm_gamma_positive --fuse_activation_function --fuse_instnorm --replace_cw_mul_add_with_depthwise_conv --remove_fakequant --remove_quantdequant --remove_redundant_quantize --remove_redundant_reshape --remove_redundant_transpose --remove_unnecessary_reshape --remove_unnecessary_slice --remove_unnecessary_strided_slice --remove_unnecessary_split --replace_non_const_fc_with_batch_matmul --resolve_customop_add --resolve_customop_batchmatmul --resolve_customop_matmul --resolve_customop_max_pool_with_argmax --shuffle_weight_to_16x1float32 --substitute_pack_to_reshape --substitute_padv2_to_pad --substitute_splitv_to_split --substitute_squeeze_to_reshape --substitute_strided_slice_to_reshape --substitute_transpose_to_reshape --transform_min_max_to_relu6 --transform_min_relu_to_relu6
```

**Comment by YongseopKim** [comment]
PTAL @mhs4670go and @seanshpark

**Comment by mhs4670go** [comment]
> ```
> $ cat mycfg.cfg
> [onecc]
> one-optimize=True
> 
> [one-optimize]
> input_path = inception_v3_tflite.circle
> output_path = inception_v3_tflite.opt.circle
> O = DEFAULT
> ```
> 
> and then, run `onecc -C mycfg.cfg`
> 
> ```
> $ cat inception_v3_tflite.opt.circle.log
> /home/dragon/Works/ONE/build/install/bin/circle2circle inception_v3_tflite.circle inception_v3_tflite.opt.circle --convert_nchw_to_nhwc --expand_broadcast_const --nchw_to_nhwc_input_shape --nchw_to_nhwc_output_shape --fold_add_v2 --fold_cast --fold_dequantize --fold_dwconv --fold_gather --fold_sparse_to_dense --forward_reshape_to_unaryop --fuse_add_with_tconv --fuse_add_with_fully_connected --fuse_batchnorm_with_conv --fuse_batchnorm_with_dwconv --fuse_batchnorm_with_tconv --fuse_bcq --fuse_preactivation_batchnorm --fuse_mean_with_mean --fuse_transpose_with_mean --make_batchnorm_gamma_positive --fuse_activation_function --fuse_instnorm --replace_cw_mul_add_with_depthwise_conv --remove_fakequant --remove_quantdequant --remove_redundant_quantize --remove_redundant_reshape --remove_redundant_transpose --remove_unnecessary_reshape --remove_unnecessary_slice --remove_unnecessary_strided_slice --remove_unnecessary_split --replace_non_const_fc_with_batch_matmul --resolve_customop_add --resolve_customop_batchmatmul --resolve_customop_matmul --resolve_customop_max_pool_with_argmax --shuffle_weight_to_16x1float32 --substitute_pack_to_reshape --substitute_padv2_to_pad --substitute_splitv_to_split --substitute_squeeze_to_reshape --substitute_strided_slice_to_reshape --substitute_transpose_to_reshape --transform_min_max_to_relu6 --transform_min_relu_to_relu6
> ```



FYI, Setting `O` key to `one-optimize` section works but that is not recommended way. Because it is not for `one-optimize` only as I said the other day. "optimization" will be able to applied to other one-cmds tools as well as `one-optimize`.

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/one-cmds/CMakeLists.txt (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/CMakeLists.txt:43]: please move this to separate PR

File: compiler/one-cmds/optimization/ODEFAULT.cfg [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# These options copied from 'onelib/constant.py`
+[one-optimize]
+; Experimental: This will convert NCHW operators to NHWC under the assumption that input model is NCHW.
+convert_nchw_to_nhwc=True
+; expand broadcastable constant node inputs
+expand_broadcast_const=True
+; onvert the input shape of the model (argument for convert_nchw_to_nhwc)
+nchw_to_nhwc_input_shape=True
+; convert the output shape of the model (argument for convert_nchw_to_nhwc)
+nchw_to_nhwc_output_shape=True
```
Comment (seanshpark) [inline compiler/one-cmds/optimization/ODEFAULT.cfg:10]: These are used when source is ONNX file. I don't think these should be in `DEFAULT`.
I wonder what meaning of `DEFAULT` is...

**Comment by seanshpark** [comment]
@YongseopKim , I've fired #9191 for discussion.

**Comment by YongseopKim** [comment]
I'd like to close this first and then discuss more. I thought it was too naive. 😢

**Pull request closed without merging**